backend/api/Services/MissionSchedulingService.cs

```diff
@@ -114,6 +114,9 @@ or MissionRunNotFoundException
                 );
                 await missionRunService.SetMissionRunToFailed(missionRun.Id, $"Mission run '{missionRun.Id}' was not started successfully. '{ex.Message}'");
             }
+            catch (RobotBusyException)
```

Comment:
In this case the mission remains on the queue, as the robot has become busy in another thread

Reply:
Resolved in conversation. We will allow this for now, but in the future this will be handled by moving the queue to ISAR